docs/components/concepts/process-instance-migration.md

```diff
@@ -370,7 +370,7 @@ The following limitations exist that may be supported in future versions:
   - An element that becomes nested in a newly added subprocess
   - An element that was nested in a subprocess is no longer nested in that subprocess
 - Mapping instructions cannot change the element type
-- Mapping instructions cannot change the task implementation, e.g. from a job worker user task to a Zeebe user task
+- Mapping instructions cannot change the task implementation, e.g. from a job worker user task to a job worker-based user task managed by Camunda, also known as a Camunda user task
```

Comment:
```suggestion
- Mapping instructions cannot change the task implementation, for example, from a job worker user task to a job worker-based user task managed by Camunda (also known as a Camunda user task)
```

Reply:
This fragment is strictly related to [my comment above](https://github.com/camunda/camunda-docs/pull/4723#issuecomment-2548565620). I think it's fine if we just remove the part in partheses:

```suggestion
- Mapping instructions cannot change the task implementation, for example, from a job worker user task to a Camunda user task.
```